unit-tests/src/test/scala/java/lang/ShortTest.scala

```diff
@@ -0,0 +1,188 @@
+package java.lang
```

Comment:
`Short` did not seem to be tested at all. Caught a few small bugs doing the tests